### PR TITLE
ramips: MT7620 Add support for Bolt BL201 board

### PR DESCRIPTION
**Pull request by yHpgi**
Manufacturer: Bolt
Model: BL201
CPU: MT7620A
RAM: DDR2 64MB
ROM: NAND 16MB
Switch: MT7620
WIFI 5G: MT7612E

### CODE REVIEW

File: target/linux/ramips/image/mt7620.mk [hunk 1]
```diff
@@ -148,6 +148,16 @@ define Device/bdcom_wap2100-sk
 endef
 TARGET_DEVICES += bdcom_wap2100-sk
 
+define Device/bolt_bl201
+  SOC := mt7620a
+  IMAGE_SIZE := 15872k
+  DEVICE_VENDOR := Bolt
+  DEVICE_MODEL := BL201
+  DEVICE_PACKAGES := kmod-mt76x2
+  SUPPORTED_DEVICES += bolt_bl201
```
Comment (1715173329) [inline target/linux/ramips/image/mt7620.mk:157]:
```suggestion
```

looks like nothing refers to it

**Comment by yHpgi** [comment]
> btw can you send this patch to upstream OpenWrt?

Someone already opens a pull request in OpenWrt a few months ago. but it's outdated and he never update his fork with the latest master branch. Is it ok if I open a pull request there?

File: target/linux/ramips/image/mt7620.mk (hunk 1 above)
Comment (yHpgi) [reply target/linux/ramips/image/mt7620.mk:157]: Ah, I see.

**Comment by yHpgi** [comment]
> > Someone already opens a pull request in OpenWrt a few months ago. but it's outdated and he never update his fork with the latest master branch. Is it ok if I open a pull request there?
> 
> Oh I saw that PR, try to ask the author do a rebase. And if he doesn't respond then you can create a new PR for it.

Ah, Got it. Thank You.

**Comment by 1715173329** [comment]
Merged, thanks!

**Comment by yHpgi** [comment]
Thank you.

**Pull request merged**